### ISSUE TITLE
 sqlite3.OperationalError: too many SQL variables

### ISSUE THREAD

**Issue by Ali-Jakhar** [langchain-ai/langchain#10321]
### Issue you'd like to raise.

I have pdf which has more than 2000 pages. file read successfully, but vector db was not created and getting this error. please suggest any solution. 

**complete error:** 
embeddings
    results = cur.execute(sql, params).fetchall()
sqlite3.OperationalError: too many SQL variables

**Comment by Ali-Jakhar** [comment]
> `TencentVectorDB`

okay, but I'm using Chromadb.

**Comment by philippe2803** [comment]
@Ali-Jakhar this is not an error from langchain but a limit set in sqlite. You can find more details on this issue here: https://stackoverflow.com/questions/60968455/python-sql-code-error-sqlite3-operationalerror-too-many-sql-variables

**Comment by Ali-Jakhar** [comment]
> @Ali-Jakhar This is not an error from langchain but a limit set in sqlite. You can find more details on this issue here: https://stackoverflow.com/questions/60968455/python-sql-code-error-sqlite3-operationalerror-too-many-sql-variables

Okay, but the error occurred when I used Chromadb to store my documents. How can I manage chunks in Chromadb to save in the sqlit? 
here is the code that I'm using,:
from langchain.vectorstores import Chroma

db = Chroma.from_documents(doc, embeddings,collection_name=collection_name,persist_directory=db_path)

and I'm getting an error message on this line.

As you see I did not use any external sqlite module to create vector db. langchain is responsible for saving vectordb.

My concern is how I can save data into sqlite If I exceed the limit using langchain? Is there any parameter that needs to be passed or something else?
